includes/admin/load.php

```diff
@@ -304,7 +304,7 @@ function perflab_install_activate_plugin_callback() {
 
 	$result = activate_plugin( $plugin_basename );
 	if ( is_wp_error( $result ) ) {
-		wp_die( esc_html( $result->get_error_message() ) );
+		wp_die( wp_kses_post( $result->get_error_message() ) );
```

Comment:
This fixes HTML mistakenly being escaped:

![image](https://github.com/WordPress/performance/assets/134745/cd385921-baff-409a-b8bf-4b8d4688f692)

Now:

![image](https://github.com/WordPress/performance/assets/134745/051f7154-9bb4-4cd5-9895-1628c41961ed)